util.py

```diff
@@ -102,3 +104,18 @@ def loadYaml(yaml_file):
 	with open(yaml_file) as fileobj:
 		return yaml.safe_load(fileobj)
 
+def getFqdn():
+	'''Gets fqdn that always ends with /'''
+	name = socket.getfqdn()
+	if not name.endswith('/'):
+		name += '/'
+	return name
+
+def getFullLink(*args):
+	'''Takes a relative link or multiple pieces of a relative link and returns
+	a full link containining the fqdn'''
+	# Parsing it into a new list, because you cannot edit args directly.
+	parts = []
+	for arg in args:
+		parts.append(str(arg))
```

Comment:
Just noticed this: appending multiple things together here without putting a `/` between them might be a problem. I'm undecided if we should do that or not (as it isn't very important to our current use)

Reply:
It's not. I'm making a list here of string versions so I can feed them all safely into posixpath.join. Which is a function from the standard libs specifically made to add together paths and add `/` if they would be missing when the path is added up.

It will become apparent once I've made the tests.